telegram/chat.py

```diff
@@ -56,6 +59,11 @@ class Chat(TelegramObject):
         sticker_set_name (:obj:`str`): Optional. For supergroups, name of Group sticker set.
         can_set_sticker_set (:obj:`bool`): Optional. :obj:`True`, if the bot can change group the
             sticker set.
+        linked_chat_id (:obj:`int`): Optional. Unique identifier for the linked chat, i.e. the
+            discussion group identifier for a channel and vice versa; for supergroups and channel
+            chats. Returned only in :meth:`telegram.Bot.get_chat`.
+        location (:class:`telegram.ChatLocation`): Optional. For supergroups, the location to which
```

Comment:
That 'Optional' should be inside the brackets right? Same for bio.

Reply:
Only for the arguments, not for the attributes. so in the change below, but thanks for the catch :)